recipes/spirv-tools/all/conandata.yml

```diff
@@ -2,3 +2,6 @@ sources:
   "v2019.2":
     sha256: 1fde9d2a0df920a401441cd77253fc7b3b9ab0578eabda8caaaceaa6c7638440
     url: https://github.com/KhronosGroup/SPIRV-Tools/archive/v2019.2.tar.gz
+  "v2020.3":
```

Comment:
Probably to change now, but regularly we remove the `v` prefix if there is one

Reply:
It was discussed here when adding this recipe: https://github.com/conan-io/conan-center-index/pull/1349#issuecomment-627818533